spec/computer_random_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "spec_helper"
+describe ComputerRandom do
+  describe "setup" do
+    # let (:andrew) {andrew = Player.new("x")}
+    let (:board) {board = Board.new}
+    let (:hal) {hal = ComputerRandom.new}
+    it "exists and takes left-over type" do
+      andrew = Player.new("x")
+
+      expect(hal).to be_a(ComputerRandom)
+
+      expect(hal.type).to eq("O")
+      expect(hal.enemy).to eq("X")
+    end
+  end
+
+  describe "place piece" do
+    it "can place a piece at a random spot" do
+      board.setup_game
+      andrew = Player.new("x")
+      hal.place_piece(board)
+      hal.place_piece(board)
+      hal.place_piece(board)
+      hal.place_piece(board)
+      hal.place_piece(board)
+      board.render_board
+    
+
+      expect(board.play_area[:A].count("O")).to (be < 5)
```

Comment:
I could be reading it wrong, but I think it's possible for hal to drop 5 "O" in the same column, just highly unlikely. We might consider something like expect(board.play_area.values.flatten.count("O")).to eq(5)

Reply:
That makes sense. I also forgot that they are objects not strings. That is a much simpler test and probable more accurate as 0 is less than 5 also. It could repeat itself every 1 in 16807 times.